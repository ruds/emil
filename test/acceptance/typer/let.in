let
  val a = 1
  val b = "foo"
in
  (a, b, 20)
end;
let
  val a = 1
  val b = "foo"
in
  a;
  (b, it)
end;
let
  val a = 1
  val b = "foo"
in
  (a, b, 20)
end : bigint * string * bigint;
let
  val a = 1
  val b = "foo"
in
  (a, b, 20)
end : bigint * string;
