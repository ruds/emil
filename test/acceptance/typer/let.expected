@0001:
val it : (bigint * string * bigint)
@0007:
val it : (string * (bigint * string * bigint))
@0014:
val it : (bigint * string * bigint)
@0020: ERROR
@0026:
